build/artifacts.props

```diff
@@ -0,0 +1,239 @@
+    <PackageArtifact Include="Microsoft.Extensions.Configuration.DockerSecrets" Category="noship" />
+    <PackageArtifact Include="Microsoft.Extensions.Configuration.EnvironmentVariables" Category="ship" Metapackage="true" />
+    <PackageArtifact Include="Microsoft.Extensions.Configuration.FileExtensions" Category="ship" Metapackage="true" />
+    <PackageArtifact Include="Microsoft.Extensions.Configuration.Ini" Category="ship" Metapackage="true" />
+    <PackageArtifact Include="Microsoft.Extensions.Configuration.Json" Category="ship" Metapackage="true" />
+    <PackageArtifact Include="Microsoft.Extensions.Configuration.UserSecrets" Category="ship" Metapackage="true" />
+    <PackageArtifact Include="Microsoft.Extensions.Configuration.Xml" Category="ship" Metapackage="true" />
+    <PackageArtifact Include="Microsoft.Extensions.CopyOnWriteDictionary.Sources" Category="noship" />
+    <PackageArtifact Include="Microsoft.Extensions.DependencyInjection" Category="ship" Metapackage="true" />
+    <PackageArtifact Include="Microsoft.Extensions.DependencyInjection.Abstractions" Category="ship" Metapackage="true" />
+    <PackageArtifact Include="Microsoft.Extensions.DependencyInjection.Specification.Tests" Category="ship" />
+    <PackageArtifact Include="Microsoft.Extensions.DiagnosticAdapter" Category="ship" Metapackage="true" />
+    <PackageArtifact Include="Microsoft.Extensions.FileProviders.Abstractions" Category="ship" Metapackage="true" />
+    <PackageArtifact Include="Microsoft.Extensions.FileProviders.Composite" Category="ship" Metapackage="true" />
+    <PackageArtifact Include="Microsoft.Extensions.FileProviders.Embedded" Category="ship" Metapackage="true" />
+    <PackageArtifact Include="Microsoft.Extensions.FileProviders.Physical" Category="ship" Metapackage="true" />
+    <PackageArtifact Include="Microsoft.Extensions.FileSystemGlobbing" Category="ship" Metapackage="true" />
+    <PackageArtifact Include="Microsoft.Extensions.HashCodeCombiner.Sources" Category="noship" />
+    <PackageArtifact Include="Microsoft.Extensions.Hosting.Abstractions" Category="ship" Metapackage="true" />
+    <PackageArtifact Include="Microsoft.Extensions.Hosting" Category="ship" Metapackage="true" />
+    <PackageArtifact Include="Microsoft.Extensions.Identity.Core" Category="ship" Metapackage="true" />
+    <PackageArtifact Include="Microsoft.Extensions.Identity.Stores" Category="ship" Metapackage="true" />
+    <PackageArtifact Include="Microsoft.Extensions.Localization" Category="ship" Metapackage="true" />
+    <PackageArtifact Include="Microsoft.Extensions.Localization.Abstractions" Category="ship" Metapackage="true" />
+    <PackageArtifact Include="Microsoft.Extensions.Logging" Category="ship" Metapackage="true" />
+    <PackageArtifact Include="Microsoft.Extensions.Logging.Abstractions" Category="ship" Metapackage="true" />
+    <PackageArtifact Include="Microsoft.Extensions.Logging.Analyzers" Category="noship" Metapackage="true" />
```

Comment:
noship, exclude

Reply:
Hold on, this is marked as noship, include in packages.csv? @pakrym I'm assuming this was a typo and should be noship, exclude?

Reply:
Good catch. I just copied what was in packages.csv. I didn't validate the Metapackages="true" attribute against what is actually in the metapackage at the moment.

Reply:
Yeah, noship exclude.